src/App.tsx

```diff
@@ -1,25 +1,29 @@
 import React from 'react';
-import logo from './logo.svg';
+//import logo from './logo.svg';
 import './App.css';
 
+import { sampleList } from './sampleList';
+
+function ListDisplay() {
```

Comment:
I would suggest renaming this component to something such as `TodoContainer` to make it's clearer what the component is about